src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -1,16 +1,15 @@
-@echo off
+@if "%_echo%" neq "on" echo off
 setlocal
 
 set PROJECT_DIR=%~1
 set DOTNET_CMD=%~2
 set TOOLRUNTIME_DIR=%~3
-set PACKAGES_DIR=%4
+set PROJECT_JSON_FILE=%4
```

Comment:
I don't want to have to change the arguments we pass to init-tools.cmd as it will break all current usages. I thought we were still going to use the toolruntime\project.json for this publishing and not the actual buildtools package.